README.md

```diff
@@ -26,6 +34,20 @@ Workflow engine using [WDL](https://github.com/broadinstitute/wdl/blob/wdl2/SPEC
 
 <!---toc end-->
 
+## Requirements
+
+The following is the toolchain used for development of Cromwell.  Other versions may work, but these are recommended.
+
+* [Scala 2.11.6](http://www.scala-lang.org/news/2.11.6)
+* [SBT 0.13.8](https://github.com/sbt/sbt/releases/tag/v0.13.8)
+* [Java 8](http://www.oracle.com/technetwork/java/javase/overview/java8-2100321.html)
+
+## Building
+
+`sbt assembly` should run the tests and create a JAR in `target/scala-2.11/`
+
+Note that the tests do require Docker to be running.  To test this out while downloading the Ubuntu image that is required for tests, run `docker pull ubuntu:latest` prior to running `sbt assembly`
+
```

Comment:
Per comment above this is no longer the case, but users would still need Docker anyway to run Dockerized workflows.